package.json

```diff
@@ -80,10 +80,10 @@
     "fast-xml-parser": "3.12.0",
     "fs-extra": "7.0.0",
     "get-installed-path": "4.0.8",
-    "gh-got": "7.1.0",
+    "gh-got": "8.0.1",
```

Comment:
I believe we no longer rely on passing `GITHUB_ENDPOINT` to `gh-got` however we do pass `opts.endpoint`, which needs to change to `opts.baseUrl`.

Reply:
yes, we still pass `opts.endpoint`, it's actually a good idea to rename it to `baseUrl`